docs/proto/docs.md

```diff
@@ -0,0 +1,398 @@
+# Protocol Documentation
+<a name="top"></a>
+
```

Comment:
Why the modification on the proto docs ? It looks like the file was commited by mistake or is from an old version

Reply:
Damn, looks like a mistake. I'll try to fix

Reply:
Removed that commit. For some reason GitHub is saying I re-added all the other commits again :shrug: